src/Lib.hs

```diff
@@ -0,0 +1,6 @@
+module Lib
+    ( someFunc
+    ) where
+
+someFunc :: IO ()
+someFunc = putStrLn "someFunc"
```

Comment:
The idea here is to have everything in a library and only have a client `main` in `app/Main.hs`.
This approach lets you reuse most of the codebase, i.e. one app lets you play Tic Tac Toe vs yourself and another one against AI.